integration/shared/isolated/enable_service_access_command_test.go

```diff
@@ -0,0 +1,321 @@
+package isolated
+
+import (
+	"code.cloudfoundry.org/cli/integration/helpers"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gbytes"
+	. "github.com/onsi/gomega/gexec"
+)
+
+var _ = Describe("enable service access command", func() {
+	Describe("help", func() {
+		When("--help flag is set", func() {
+			It("displays command usage to output", func() {
+				session := helpers.CF("enable-service-access", "--help")
+				Eventually(session).Should(Say("NAME:"))
+				Eventually(session).Should(Say("\\s+enable-service-access - Enable access to a service or service plan for one or all orgs"))
+				Eventually(session).Should(Say("USAGE:"))
+				Eventually(session).Should(Say("\\s+cf enable-service-access SERVICE \\[-p PLAN\\] \\[-o ORG\\]"))
+				Eventually(session).Should(Say("OPTIONS:"))
+				Eventually(session).Should(Say("\\s+\\-o\\s+Enable access for a specified organization"))
+				Eventually(session).Should(Say("\\s+\\-p\\s+Enable access to a specified service plan"))
+				Eventually(session).Should(Say("SEE ALSO:"))
+				Eventually(session).Should(Say("\\s+marketplace, service-access, service-brokers"))
+				Eventually(session).Should(Exit(0))
+			})
+		})
+	})
+
+	When("no service argument was provided", func() {
+		It("displays a warning, the help text, and exits 1", func() {
+			session := helpers.CF("enable-service-access")
+			Eventually(session.Err).Should(Say("Incorrect Usage: the required argument `SERVICE` was not provided"))
+			Eventually(session).Should(Say("NAME:"))
+			Eventually(session).Should(Say("\\s+enable-service-access - Enable access to a service or service plan for one or all orgs"))
+			Eventually(session).Should(Say("USAGE:"))
+			Eventually(session).Should(Say("\\s+cf enable-service-access SERVICE \\[-p PLAN\\] \\[-o ORG\\]"))
+			Eventually(session).Should(Say("OPTIONS:"))
+			Eventually(session).Should(Say("\\s+\\-o\\s+Enable access for a specified organization"))
+			Eventually(session).Should(Say("\\s+\\-p\\s+Enable access to a specified service plan"))
+			Eventually(session).Should(Say("SEE ALSO:"))
+			Eventually(session).Should(Say("\\s+marketplace, service-access, service-brokers"))
+			Eventually(session).Should(Exit(1))
+		})
+	})
+
+	When("two services arguments are provided", func() {
+		It("displays an error, and exits 1", func() {
+			session := helpers.CF("enable-service-access", "a-service", "another-service")
+			Eventually(session).Should(Say("FAILED"))
+			Eventually(session.Err).Should(Say(`Incorrect Usage: unexpected argument "another-service"`))
+			Eventually(session).Should(Say("NAME:"))
+			Eventually(session).Should(Say("\\s+enable-service-access - Enable access to a service or service plan for one or all orgs"))
+			Eventually(session).Should(Say("USAGE:"))
+			Eventually(session).Should(Say("\\s+cf enable-service-access SERVICE \\[-p PLAN\\] \\[-o ORG\\]"))
```

Comment:
This might be a good candidate for regexp.QuoteMeta. Not merge-blocking though.

Reply:
Oh, just noticed you have a `\s+` at the beginning, so QuoteMeta won't fit if that's needed. Well, you could remove one level of `\` by using \` instead of `"`, but I don't feel strongly about this.